src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int LOTTERY_COUNTER = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery1 = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
avoid numbers in variable names